addons/hr_holidays/security/hr_holidays_security.xml

```diff
@@ -135,7 +135,7 @@
     <record id="hr_leave_rule_multicompany" model="ir.rule">
         <field name="name">Time Off: multi company global rule</field>
         <field name="model_id" ref="model_hr_leave"/>
-        <field name="domain_force">['|', ('holiday_status_id.company_id', '=', False), ('holiday_status_id.company_id', 'in', company_ids)]</field>
+        <field name="domain_force">[('company_id', 'in', company_ids)]</field>
```

Comment:
This is noupdate record, needs upgrade script

Reply:
This is actually part of another PR I have going on, on which this one is rebased, but it is of course a valid comment for that one: https://www.odoo.com/web#id=3383478&cids=1&menu_id=4720&action=333&active_id=1251&model=project.task&view_type=form
I will create an upgrade script for that task as well then.